settlement/config.go

```diff
@@ -1,17 +1,14 @@
 package settlement
 
-import "github.com/ignite/cli/ignite/pkg/cosmosaccount"
-
 // Config for the DymensionLayerClient
 type Config struct {
-	KeyringBackend cosmosaccount.KeyringBackend `json:"keyring_backend"`
```

Comment:
why did you remove `KeyringBackend`?

Reply:
we only support `test` for the sequencer
no reason to expose it as a configuration parameter if it's not supported

Reply:
are you sure? In the cosmosclient we init I see this keyring backends supported:
```
type KeyringBackend string

const (
	// KeyringTest is the test keyring backend. With this backend, your keys will be
	// stored under your app's data dir,
	KeyringTest KeyringBackend = "test"

	// KeyringOS is the OS keyring backend. with this backend, your keys will be
	// stored in your operating system's secured keyring.
	KeyringOS KeyringBackend = "os"

	// KeyringMemory is in memory keyring backend, your keys will be stored in application memory.
	KeyringMemory KeyringBackend = "memory"
)
```

either way we'll for sure need to support more keyring backends in the near future (specifically ledger) so not sure what's the motivation for removing it

Reply:
yeah but the sequencer can't use anything else as it will request signing approval for each submission.
anyway I leave it in the configuration